ovn-tester/ovn_workload.py

```diff
@@ -140,19 +142,29 @@ def configure_localnet(self, physical_net):
         self.run(cmd=f'ovs-vsctl -- set open_vswitch . '
                  f'external-ids:ovn-bridge-mappings={physical_net}:br-ex')
 
+    def configure_external_host_net(self, ip, plen, gw):
+        if ip and plen:
+            cmd = f'ip netns exec ext-ns ip addr add {ip}/{plen} dev veth0'
+            self.run(cmd=cmd)
```

Comment:
Maybe running the command at the end in a single ssh transaction?

Reply:
@almusil ack, I will fix it